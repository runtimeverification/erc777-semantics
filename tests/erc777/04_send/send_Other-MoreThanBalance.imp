test.preamble();

// Testing send(To, Value, "") when To != caller and Value > BalanceFrom.

test.send(4, 71, "");  // balance of caller is 70
