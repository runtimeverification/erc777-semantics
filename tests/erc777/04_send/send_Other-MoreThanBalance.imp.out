7 is sending 71 to 4 (data:)
REVERT
Balance of 7 is 70
Balance of 4 is 40
